src/unitxt/processors.py

```diff
@@ -277,3 +279,57 @@ def process_value(self, text: Any) -> Any:
 
         except:
             return 0
+
+
+class InferDictsToBinaryLogprobs(FieldOperator):
+    binary_class_names: list[str]
+
+    def process_value(self, obj: Any) -> Any:
+        try:
+            n_first_tokens = np.min([3, len(obj)])
+        except:
+            return []
+
+        for i in range(n_first_tokens):
+            try:
+                pos_probs, neg_probs = self.get_pos_neg_probs(pred_dict=obj[i])
+                if pos_probs or neg_probs:
+                    sum_probs = sum(pos_probs) + sum(neg_probs)
+                    if sum_probs > 0.0001:
+                        return sum(pos_probs) / sum_probs
+            except:
+                pass
+        return []
```

Comment:
We should decide what this post processor return.  I think it's the probably of the positive class.
So if it's not able to find anything it should return np.nan.

Reply:
done